v4m-container/README.md

```diff
@@ -21,15 +28,18 @@ cd v4m-container
 docker build --no-cache -t v4m .
 ```
 
-The Docker image `v4m` will contain Helm, kubectl, and other executables needed to run the v4m deployment.
+The Docker image `v4m` will contain Helm, kubectl, and other executables needed to run the Viya Monitoring for Kubernetes deployment.
 
-#### Kubeconfig File
+### Kubeconfig File
 
 To ensure that your kubeconfig files are available from within the Docker container, you can copy the kubeconfig files to the `./v4m-container/kubeconfig` directory or mount the file or directory during your `docker run` command.
 
-#### (Optional) Custom Deployment Configurations
+### (Optional) Custom Deployment Configurations
 
-If you have made changes using a USER_DIR directory, you can copy the contents of this directory to the `./v4m-container/user_dir` directory or mount the file or directory during your `docker run` command.
+You can customize the Viya 4 Monitoring for Kubernetes deployment by editing files in a USER_DIR directory. See the [monitoring README](../monitoring/README.md) and [logging README](../logging/README.md)
+) for detailed information about the customization process and about determining valid customization values.
+
+If you make any customization, you can place the files in the provided `./v4m-container/user_dir` directory to make them available within the container.  If you want to update your custom deployment configuration, you will need to update the contents of the `./v4m-container/user_dir` directory and rebuild the Docker container.
 
```

Comment:
Hmmm, so putting files in either ./v4m-container/kubeconfigs or ./v4m-container/user_dir subdirectories results in their contents being "built in" to the container.  I think that's an important detail we probably need to call out more prominently .  It also seems like a tradeoff one has to make/accept if they want to using this approach (i.e. working "inside" the container) instead of the other approach.  With the other approach, the contents of the two sub-directories are "live" correct?  That is, changes to files in those subdirectories can be made after submitting a docker run command and have those changes picked up for subsequent call to docker run without having to re-running the docker build command.